README.md

```diff
@@ -15,6 +15,25 @@ This version will look for an `app_config.yml` file on `config` folder in the pr
 - Add option for server config, right now only creates nginx vhost file and mysql database files for rails AR projects.
 - More stuff
 
+# Use
+In a rails app, simply include the gem in your Gemfile:
+```
+gem 'biran'
+```
+
+In a non-rails app, you will need to do some extra work to make the tasks available to rake. In your Rakefile you will need to manually include things.
+Here is a minimal Rakefile for a basic ruby app that includes the biran tasks along with any local tasks in `lib/tasks`
+```
+require 'bundler/setup'
+require 'biran'
+
+spec = Gem::Specification.find_by_name 'biran'
```

Comment:
is spec the right name?

Reply:
lol... I just copied from your example. Open to naming suggestions

Reply:
cargo cult x 2, yeah I got this from a sample including rspec... so that is why `spec` made sense...

Reply:
😊   changed then